src/App.js

```diff
@@ -1,27 +1,46 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import Plus from './templates/add.svg';
+import { isEmpty } from 'lodash';
+import classNames from 'classnames';
 
 import Title from './components/Title';
 import Input from './components/Input';
 import Button from './components/Button';
 import List from './components/List';
 import Counter from './components/Counter';
+import { addTask, removeTask } from './store/actions/listActions';
 
 class App extends Component {
   state = {
-    tasks: ['something'],
     value: '',
+    isInputEmpty: false,
+    isValueExist: false,
+  }
+  addTask = () =>{
+    const { value } = this.state
+    if (isEmpty(value)) {
+      this.setState({isInputEmpty: true})
+      return null
+    } else if (this.props.tasks.includes(value)) {
```

Comment:
we've talked about destructuring. there is state destructured, plase destructure also `props`